faceUnlocker.py

```diff
@@ -0,0 +1,105 @@
+import cv2
```

Comment:
this file should be in face unlock branch/pull request, Why changes are here?

Reply:
I did them in Serialized way.